tools/rosbag/src/rosbag/rosbag_main.py

```diff
@@ -449,6 +452,7 @@ def fix_cmd(argv):
                     
         print('Try running \'rosbag check\' to create the necessary rule files or run \'rosbag fix\' with the \'--force\' option.')
         os.remove(outname)
+	sys.exit(1)
```

Comment:
The indentation of this line uses a tab which isn't allowed in Python 3. Please change this to spaces.